app/scripts/directives/resources.js

```diff
@@ -16,6 +16,53 @@ angular.module('openshiftConsole')
       }
     };
   })
+  .directive('podStatus', function($filter) {
+    return {
+      restrict: 'E',
+      scope: {
+        podStatus: '=',
+        initStatus: '=?',
```

Comment:
We should decide if we want one directive for all statuses or a directive for each individual container status or (or both). I'm OK either way. If we have one directive, I'd call it `containerStatuses` since it doesn't show all of pod status, just the status of the containers.

If we have one directive, I'd remove `initStatus` and just always show it. @jwforres and I talked about maybe collapsing the init container status with a summary when the init containers are complete. @cdcabrera let's discuss